js/google_ima.js

```diff
@@ -299,6 +301,11 @@ require("../html5-common/js/utils/utils.js");
         {
           _IMA_SDK_tryInitAdContainer();
         }
+        else if (!_IMAAdsLoader)
```

Comment:
Is this an `else if` because `_IMA_SDK_tryInitAdContainer` also creates the Ads Loader?